test/test_disc_quantile.jl

```diff
@@ -0,0 +1,15 @@
+@test DiscretizeQuantile(2) == DiscretizeQuantile(2, false)
```

Comment:
Thank you for adding tests!

It may not be covered by the other methods, but could you please test [1,1,1] and verify that we throw an error?